parm/config/config.efcs

```diff
@@ -56,15 +56,18 @@ export restart_interval=${restart_interval:-6}
 # For IAU, write restarts at beginning of window also
 if [ $DOIAU_ENKF = "YES" ]; then
     export restart_interval="6 -1"
-    if [[ "$SDATE" = "$CDATE" ]]; then export restart_interval="3 -1"; fi
+    if [[ "$SDATE" = "$CDATE" && $EXP_WARM_START = ".false." ]]; then export restart_interval="3 -1"; fi # Cold starting
 fi
 
 export OUTPUT_FILETYPES="$OUTPUT_FILE"
 if [[ "$OUTPUT_FILE" == "netcdf" ]]; then
     export  ichunk2d=0; export jchunk2d=0
     export  ichunk3d=0; export jchunk3d=0;  export kchunk3d=0
     RESTILE=`echo $CASE_ENKF |cut -c 2-`
-    if [[ "$machine" == "WCOSS_DELL_P3" ]] || [[ "$machine" == "WCOSS2" ]]; then
+    if [[ "$machine" == "WCOSS2" ]]; then
```

Comment:
This section could use a bit of clean-up.  There is just too much duplication in the if-blocks that can be pulled out.

Reply:
cleaned up and consolidated, please see updates